src/components/ResultsCount.tsx

```diff
@@ -14,8 +14,8 @@ import { useComposedCssClasses } from '../hooks/useComposedCssClasses';
  * @public
  */
 export interface ResultsCountCssClasses {
-  resultCountText?: string,
-  resultCountText___loading?: string
+  resultCountContainer?: string,
```

Comment:
nit: can these be renamed to `resultsCountContainer` and `resultsCount___loading` (add "s" after "result") to match the component name?

Reply:
done!